dotCMS/src/main/java/com/dotcms/storage/ObjectWriterDelegate.java

```diff
@@ -0,0 +1,19 @@
+package com.dotcms.storage;
+
+import java.io.OutputStream;
+import java.io.Serializable;
+
+/**
+ * Simple method to write an object into the stream
+ * @author jsanca
+ */
+@FunctionalInterface
+public interface ObjectWriterDelegate {
+
+    /**
+     * Writes the object into the stream
+     * @param stream {@link OutputStream}
+     * @param object {@link Serializable}
+     */
+    public void write (final OutputStream stream, Serializable object);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'write': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)